client/wfm/src/result/StepResult.ts

```diff
@@ -0,0 +1,35 @@
+import { STATUS } from '../status';
+import { Step } from '../step/Step';
+
+/**
+ * Output of a single {@link Step} in the context of a {@link WorkOrder}
+ */
+export interface StepResult {
+  /**
+   * Form data output of the {@link Step}'s execution
+   */
+  submission?: {
+    [key: string]: any
+  };
+  /**
+   * Identifier of the User that submitted the form data
+   */
+  submitter?: string;
+
+  /**
+   * Unix timestamp of the creation of this result
+   */
+  timestamp: number;
```

Comment:
We may document how we store date in system later (timestamp) aproach and how to work with that in UI (using moment etc.)